src/CodeExamples.sln

```diff
@@ -23,6 +23,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMap", "HashMap\HashMap.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestHashMap", "TestHashMap\TestHashMap.csproj", "{EBDC072A-4291-4482-9BAE-2490C190B1B6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateUintCapacity", "HashMap.Test\HashMapNamespace.CalculateUintCapacity\HashMapNamespace.CalculateUintCapacity.csproj", "{4DC0AECB-322A-427A-B49B-FA54AF5A8D16}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateCapacity", "HashMap.Test\HashMapNamespace.CalculateCapacity\HashMapNamespace.CalculateCapacity.csproj", "{52CF5877-0FA0-482F-A9E8-137C1FF052EC}"
```

Comment:
### Compilation Error
```
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity (roost-1709036480) $ dotnet test
  Determining projects to restore...
  Restored /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj (in 429 ms).
  1 of 2 projects are up-to-date for restore.
  HashMap -> /workspaces/CodeExamples/src/HashMap/bin/Debug/net6.0/HashMap.dll
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(84,26): error CS0305: Using the generic type 'HashHelpers<T>' requires 1 type arguments [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(100,26): error CS0305: Using the generic type 'HashHelpers<T>' requires 1 type arguments [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(115,71): error CS0305: Using the generic type 'HashHelpers<T>' requires 1 type arguments [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(130,26): error CS0305: Using the generic type 'HashHelpers<T>' requires 1 type arguments [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(87,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(103,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(88,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(104,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(133,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/CalculateCapacity.cs(134,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity/HashMapNamespace.CalculateCapacity.csproj]
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateCapacity (roost-1709036480) $ 
```